pkgs/development/python-modules/overpy/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-runner
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "overpy";
+  version = "0.6";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "DinoTools";
+    repo = "python-overpy";
+    rev = version;
+    hash = "sha256-Tl+tzxnPASL4J6D/BYCEWhXe/mI12OVgNT5lyby3s7A=";
+  };
+
+  nativeBuildInputs = [ pytest-runner ];  # pytest-runner included in setup file
```

Comment:
Patch `pytest-runner` out in `setup.py`. It should work without it.

Reply:
Thanks @fabaff!